packages/runtime-core/src/renderer.ts

```diff
@@ -1216,6 +1216,7 @@ function baseCreateRenderer(
       // no update needed. just copy over properties
       n2.component = n1.component
       n2.el = n1.el
+      // instance.vnode = n2
     }
```

Comment:
Soultion2: This is here caused that the vnode isn't euquals to `parent.subTree` when parent is first rerender